benchmarks/smallmatvec/manual_versions/openmp/innerloop.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+output=innerloop.txt
```

Comment:
Please add a comment line explaining what this file does.

Reply:
Done